src/server/src/main/java/io/cassandrareaper/resources/ClusterResource.java

```diff
@@ -264,6 +280,7 @@ private Cluster createClusterWithSeedHost(String seedHostInput) throws ReaperExc
     return new Cluster(clusterName.get(), partitioner.get(), seedHostsFinal);
   }
 
+  @Deprecated
```

Comment:
`PUT` and `POST` endpoints are not the same thing, so this shouldn't be deprecated.

The PUT endpoint offers a number of valid differences from enforcing a safety check that the seed must belong to the named cluster, to the presumption of idempotence.

This has been addressed in #401